qsa-calculator-server/src/main/resources/messages.properties

```diff
@@ -39,9 +39,9 @@ error.positive.parameter.steps = Steps cannot be <b>negative</b> or <b>0</b>
 error.positive.parameter.Alpha = The <b>α</b> value should be higher than <b>0</b>
```

Comment:
I think all Alpha-s should change as well to Lambda. This applies to each language.